dev-support/docker/Dockerfile_windows_10

```diff
@@ -108,6 +108,12 @@ RUN powershell Copy-Item -Path "C:\LibXXHash\usr\bin\*.dll" -Destination "C:\Pro
 RUN powershell Copy-Item -Path "C:\LibZStd\usr\bin\*.dll" -Destination "C:\Program` Files\Git\usr\bin"
 RUN powershell Copy-Item -Path "C:\RSync\usr\bin\*" -Destination "C:\Program` Files\Git\usr\bin"
 
+# Install Python 3.10.11.
```

Comment:
curious any specific reason for using 3.10.11? If I see here:
https://www.python.org/ftp/python/
we have 3.12.0 as well

Reply:
I normally choose a version that's less recent so that it's mature and has got the appropriate fixes (if any that were discovered upon release).
1. `3.10.11` -> Would've gone through 11 iterations after release, hopefully containing the bug fixes from the time when `3.10.0` was released.
2. `3.12.0` -> Is just the first release and could potentially have some bugs.

This is just a practice that I follow. I don't mind bumping up the version to `3.12.0` if you insist.

Reply:
nopes, just wanted to know, if we have any challenges with the higher versions. Good with me